foss4gna_24/.ipynb_checkpoints/big-EO-viz-checkpoint.ipynb

```diff
@@ -0,0 +1,1098 @@
+{
```

Comment:
<p>This is a local file, should we demonstrating from a S3 location leveraging the COPC format? Also do we have data we can share publicly?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/geotrees-demo/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Done. Re data sharing: need to check with the UWG as this is derived norm file created by UWG.</p>